internal/ingress/controller/config/config.go

```diff
@@ -929,7 +929,7 @@ func NewDefault() Configuration {
 		GlobalRateLimitMemcachedPoolSize:       50,
 		GlobalRateLimitStatusCode:              429,
 		DebugConnections:                       []string{},
-		StrictValidatePathType:                 false, // TODO: This will be true in future releases
+		StrictValidatePathType:                 true,
```

Comment:
This breaks legitimate ingress paths which contain dots, [such as all the `/.well-known/` URLS](https://en.wikipedia.org/wiki/Well-known_URI):
 * https://en.wikipedia.org/wiki/Well-known_URI
 * https://github.com/kubernetes/ingress-nginx/issues/11176
 * https://github.com/cert-manager/cert-manager/issues/7791